jdaviz/core/tools.py

```diff
@@ -447,6 +447,29 @@ def is_visible(self):
             )
 
 
+@viewer_tool
+class SkewerSelectFootprint(CheckableTool, HubListener):
```

Comment:
instead of a separate tool, was a modification of the current tool considered (ctrl+click, keypress event, etc) considered?

Reply:
In the interest of maximizing discoverability, is it possible to have the tool menu show a selection icon with an expand arrow, which opens a submenu of footprint selection options?

Reply:
we could split that "category" into two, but that increases the probability of horizontal overflow issues (which are not currently handled well) for narrow viewers.  We also had a long-term plan (not currently prioritized, afaik) to allow activating viewer tools from the plugins themselves which I think would really help in this case as the footprint plugin could then put the tools in more context.